lib/main.dart

```diff
@@ -1,14 +1,286 @@
+              listOfChipNames: _genres,
+              listOfChipIndicesCurrentlySeclected: [0],
+              inactiveTextColorList: const [Colors.redAccent],
+              activeBgColorList: const [Colors.redAccent],
+            ),
+            //In the future the idea is that if the user click on one genre, it redirects to another page where there are all the movies related to that genre
+            const SizedBox(
+              height: 20,
+            ),
+            Text(
+              textAlign: TextAlign.justify,
+              style: const TextStyle(
+                fontFamily: app_constants.ConstantsClass.textFont,
+                fontSize: 17.0,
+                color: Colors.white,
+              ),
+              description,
+              softWrap:
+                  true, //With this, long text will not overflow the widget's bound
+            ),
+            const SizedBox(
+              height: 10,
+            ),
+          ],
+        ));
+  }
+}
+
+class _Image extends StatelessWidget {
+  const _Image({super.key});
+
+  final String imagePath = 'assets/images/poster.jpg';
 
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return ;
+    return Center(
+      child: Container(
+        height: 300,
+        width: 300,
+        decoration: BoxDecoration(
+          border: Border.all(
+            width: 2.5,
+            color: Colors.black,
+          ),
+        ),
+        child: ClipRRect(
+          //it helps to round the image
+          borderRadius: BorderRadius.circular(20), // Image border
+          child: SizedBox.fromSize(
+            size: const Size.fromRadius(48), // Image radius
+            child: Image.asset(
+              imagePath,
+              fit: BoxFit.fill,
+            ),
+          ),
+        ),
+      ),
+    );
   }
-}
\ No newline at end of file
+}
+
+class _Subtitle extends StatelessWidget {
+  final String originalTitle = 'Original Title: The Super Mario Bros. Movie',
+      dateRelease = '2023-04-05';
+
+  const _Subtitle({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            originalTitle,
+            style: const TextStyle(
+              fontFamily: app_constants.ConstantsClass.textFont,
+              fontSize: 15.0,
+              color: Colors.grey,
+              fontStyle: FontStyle.italic,
+            ),
+          ),
+          Text(
+            dateRelease,
+            style: const TextStyle(
+                fontFamily: app_constants.ConstantsClass.textFont,
+                fontSize: 15.0,
+                color: Colors.grey,
+                fontStyle: FontStyle.italic),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Title extends StatelessWidget {
+  final String movieTitle = 'The Super Mario Bros. Movie';
```

Comment:
Let's receive this title as an argument of the widget. And the same for the rest of widgets